site/source/docs/contracts_configuration.md

```diff
@@ -334,6 +334,44 @@ deploy: {
 ...
 </code></pre>
 
+## Proxy Contract Support
```

Comment:
It's not really obvious about *how* this is can be used, so maybe a contract example would help?

Also, a reference to the original issue (https://github.com/embarklabs/embark/issues/1689) might also help clarify some things.